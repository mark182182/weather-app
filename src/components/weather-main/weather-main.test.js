import React from 'react';
import { shallow, mount } from 'enzyme';
import WeatherMain from './weather-main';

it('renders without crashing', () => {
  shallow(<WeatherMain />);
});

it('should match the snapshot', () => {
  const wrapper = shallow(<WeatherMain />);
  expect(wrapper.html()).toMatchSnapshot();
});
