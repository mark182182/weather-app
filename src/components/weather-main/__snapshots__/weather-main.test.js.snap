// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `"<div class=\\"MuiContainer-root weather-container MuiContainer-maxWidthLg\\"><div class=\\"MuiGrid-root weather-main MuiGrid-container MuiGrid-direction-xs-column\\"><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-justify-xs-space-between\\"><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-6\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated\\">Location: </label><div class=\\"MuiFormControl-root MuiTextField-root\\"><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl\\"><input type=\\"search\\" aria-invalid=\\"false\\" placeholder=\\"e.g. London\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputTypeSearch\\"/><fieldset aria-hidden=\\"true\\" style=\\"padding-left:8px\\" class=\\"PrivateNotchedOutline-root-179 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legend-180\\" style=\\"width:0.01px\\"><span>&#8203;</span></legend></fieldset></div></div></div><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-3\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated\\">Radius: </label><div class=\\"MuiInputBase-root MuiOutlinedInput-root\\"><div class=\\"MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input\\" tabindex=\\"0\\" role=\\"button\\" aria-labelledby=\\" \\" aria-haspopup=\\"listbox\\">10</div><input type=\\"hidden\\" value=\\"10\\"/><svg class=\\"MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\"><path d=\\"M7 10l5 5 5-5z\\"></path></svg><fieldset aria-hidden=\\"true\\" style=\\"padding-left:8px\\" class=\\"PrivateNotchedOutline-root-179 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legend-180\\" style=\\"width:0.01px\\"><span>&#8203;</span></legend></fieldset></div></div><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-3\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated\\">Results: </label><div class=\\"MuiInputBase-root MuiOutlinedInput-root\\"><div class=\\"MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input\\" tabindex=\\"0\\" role=\\"button\\" aria-labelledby=\\" \\" aria-haspopup=\\"listbox\\">10</div><input type=\\"hidden\\" value=\\"10\\"/><svg class=\\"MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\"><path d=\\"M7 10l5 5 5-5z\\"></path></svg><fieldset aria-hidden=\\"true\\" style=\\"padding-left:8px\\" class=\\"PrivateNotchedOutline-root-179 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legend-180\\" style=\\"width:0.01px\\"><span>&#8203;</span></legend></fieldset></div></div></div><div class=\\"MuiPaper-root weather-board MuiPaper-elevation1 MuiPaper-rounded\\"><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div><div class=\\"MuiSkeleton-root MuiSkeleton-rect weather-card-skeleton MuiSkeleton-animate\\"></div></div></div></div>"`;
